cinema/models.py

```diff
@@ -35,12 +40,22 @@
         return f"{self.first_name} {self.last_name}"
 
 
+# "http://127.0.0.1:8000/media/uploads/movies/liar-93733032-c097-4a38-9b2b-20404e7186e6.jpeg"
+
+def movie_image_path(instance: "Movie", filename: str) -> Path:
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}" + pathlib.Path(
+        filename
+    ).suffix
+    return pathlib.Path("upload/movies/") / pathlib.Path(filename)
```

Comment:
The `movie_image_path` function could be simplified for better readability. Instead of using the `pathlib.Path` constructor twice, you can use the `/` operator to join paths. Here is a cleaner way to write this function: 
 ```python
 def movie_image_path(instance: 'Movie', filename: str) -> Path:
     filename = f'{slugify(instance.title)}-{uuid.uuid4()}{Path(filename).suffix}'
     return Path('upload/movies') / filename
 ```